CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+## 1.0.1 - 2021-03-12
```

Comment:
I don't like changelogs because they cause uneeded git merge conflicts, and the git log serves almost the same purpose.  However if you have  a changelog you might consider adding a .gitattributes file with a line:
CHANGELOG.md merge=union
which will mostly just take the lines from both sides when there is a conflict.

Reply:
I'm partially with you. That said: I consider change-log entries a bit more targeted towards the user. That is, often multiple commits would be summarized in one sentence.